app/views/foreman/unattended/userdata-katello.erb

```diff
@@ -16,6 +16,9 @@ manage_etc_hosts: true
 ssh_authorized_keys:
```

Comment:
it should be fixed now, I added check for either sshkey or remote_execution_ssh_keys being present